app/core/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Recipe(models.Model):
+    """Recipe object"""
+    name = models.TextField(blank=False)
```

Comment:
`blank=False` is the default so there is no real need to be explicit if you don't want so.

Reply:
Good to know

Reply:
I understood it to mean 'this field is required' but what I actually wanted was blank=True in the description property to signify an optional entry